src/deployment/deployment.service.ts

```diff
@@ -93,30 +72,31 @@ export class DeploymentService {
   private initializeDeployments(): Deployment[] {
     return [
       {
-        exchangeId: ExchangeId.OGEthereum,
-        blockchainType: BlockchainType.Ethereum,
-        rpcEndpoint: this.configService.get('ETHEREUM_RPC_ENDPOINT'),
-        harvestEventsBatchSize: 100000,
+        exchangeId: ExchangeId.OGHedera,
```

Comment:
Add Hedera deployment and remove unnecessary deployments. Each deployment contains all network informations and carbon contract addresses. Updater jobs will use these deployment when crawl data and update token price